src/chigame/users/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls import path
-
+from .views import UserListView
 from chigame.users.views import user_detail_view, user_redirect_view, user_update_view
 
 app_name = "users"
 urlpatterns = [
     path("~redirect/", view=user_redirect_view, name="redirect"),
     path("~update/", view=user_update_view, name="update"),
     path("<int:pk>/", view=user_detail_view, name="detail"),
+    path('users/', UserListView.as_view(), name='user-list'),
+    path('users/', user_list, name='user-list'),
```

Comment:
This line is redundant and causes the `python3 manage.py runserver` command to not work:

```bash
    path('users/', user_list, name='user-list'),
                   ^^^^^^^^^
NameError: name 'user_list' is not defined
```

Removing this line then lets the server run.

Reply:
The code now runs on the server and using python check. I changed the path of the api to api/users